x-pack/plugins/security/public/plugin.tsx

```diff
@@ -181,6 +185,14 @@ export class SecurityPlugin
       uiApi: getUiApi({ core }),
       navControlService: this.navControlService.start({ core, authc: this.authc }),
       authc: this.authc as AuthenticationServiceStart,
+      userProfiles: {
```

Comment:
__note:__ when/if we make `user_profile` a top-level folder we might want to create user profile service that will return a proper contract so that we don't have do `bind`s here.